Luminary099/R30.agc

```diff
@@ -246,7 +247,7 @@ DOTICK		CAF	1SEC		# RE-REQUEST TICKTEST.
 		MASK	V82FLAGS
 		INDEX	A
 		TC	+1
-		TC	TASKOVER	# IF NO FLAGBITS SET DONT' CHANGE TFF OR
+		TC	TASKOVER	# IF NO FLAGBITS SET DON'T CHANGE TFF OR
 					# -TPER, BUT CONTINUE LOOP.
 		TC	TPERTICK	# ONLY BIT 1 SET.  INCR -TPER BY 1 SEC.
 TFFTICK		CAF	1SEC		# ONLY BIT 2 SET.  INCR TFF BY 1 SEC.
```

Comment:
![image](https://user-images.githubusercontent.com/3440094/95389078-38e80a80-08eb-11eb-9594-7362025550db.png)
Line 274 (below), single space before **and** after `NOTE:`